addon-resizer/README.md

```diff
@@ -102,3 +102,146 @@ spec:
           configMap:
             name: nannyconfig
 ```
+
+## Addon resizer configuration
+
+To follow instructions in this section, set environment variable `ADDON_NAME` to
+name of your addon, for example:
+
+```
+ADDON_NAME=$ADDON_NAME
+```
+
+### Overview
+
+Some addons are scaled by Addon Resizer running as a sidecar container. The default
+configuration is passed to Addon Resizer via command-line flags and consists of
+parameters:
+
+1. CPU parameters:
+  ```
+  --cpu
+  --extra-cpu
+  ```
+
+  On n-node cluster, the total CPU assigned to the addon will be computed as:
```

Comment:
It will be at least this, possibly more, since there are buckets for multiple node counts. Same for memory below.